content_zh/docs/concepts/security/_index.md

```diff
@@ -4,3 +4,217 @@ description: 描述 Istio 的授权与认证功能。
 weight: 30
 type: section-index
 ---
+
+## 综述
+
+Istio 基于角色的访问控制（RBAC）为 Istio 网格中的服务提供命名空间级、服务级、方法级访问控制。它的特点:
+* 基于角色的语义，它简单易用。
+* 服务到服务以及用户端到服务的授权。
+* 在角色和角色绑定中可以通过自定义属性保证灵活性。
+
+## 架构
+
+下面的图表显示了 Istio RBAC 体系结构。操作者可以指定 Istio RBAC 策略。策略则保存在 Istio 配置存储中。
+
+{{< image width="80%" ratio="56.25%"
+    link="https://github.com/istio/istio.github.io/tree/master/content/docs/concepts/security/IstioRBAC.svg"
+    alt="Istio RBAC"
+    caption="Istio RBAC Architecture"
+    >}}
+
+Istio 的 RBAC 引擎做了下面两件事：
+* **获取 RBAC 策略.** Istio RBAC 引擎关注 RBAC 策略的变化。如果它看到任何更改，将会获取更新后的 RBAC 策略。
+* **授权请求.** 在运行时，当一个请求到来时，请求上下文会被传递给 Istio RBAC 引擎。RBAC 引擎根据传递的内容对环境进行评估，并返回授权结果（允许或拒绝）。
+
+### 请求上下文
+
+在当前版本中，Istio RBAC 引擎被实现为一个[Mixer 适配器](https://github.com/istio/istio.github.io/tree/master/content_zh/docs/concepts/policies-and-telemetry#adapters)。请求上下文则作为[授权模板](https://github.com/istio/istio.github.io/tree/master/content/docs/reference/config/policy-and-telemetry/templates/authorization/)的实例。请求上下文包含请求和授权模块需要环境的所有信息。特别是两个部分：
```

Comment:
https://github.com/istio/istio.github.io/tree/master/content_zh/docs/concepts/policies-and-telemetry#adapters
=>
/docs/concepts/policies-and-telemetry#adapters

Reply:
I have try relative path,but i can`t pass CI